ietf-scitt-charter.md

```diff
@@ -1,51 +1,57 @@
 
-
-A minimally, simple and concise set of building blocks that interact in a standardized way could guarantee a long term accountability and inter-operability for supply chain components throught their lifecylces across architecturally diverse systems.
+A minimal, simple and concise set of building blocks that interact in a standardized way will serve to assure long term accountability and inter-operability for supply chain components throughout their lifecycles across architecturally diverse systems.
 
 Goals
 =====
-The WG has defined a proposed architecture (link to architecture) for securing a software supply chain. The WG will standardize the overall workflow and essential building blocks that make up the architecture.
+Based on an input document on the architecture (draft-birkholz-scitt-architecture-00), the WG will: 
 
-The WG will use the existing work already done within other IETF WGs such as 
-COSE WG, IETF RATS WG, as appropriate.
+1. standardize the overall security flows for securing a software supply chain, covering the essential building blocks that make up the architecture, and
+2. specify these building blocks, employing the existing work already done within other IETF WGs such as COSE WG, IETF RATS WG, as appropriate.
+
+The input document on the architecture may be refined in the process.
 
 Non-Goals
 =========
 The WG does not:
-1. Make recommendations or suggestions on best practices on how to design the supply chain
-2. Proposing a universal/centralized registry for supply chain data.
-3. Prevent supply chain issuers from making false claims. 
-4. Any specific implementation guidance on Storage, Query or retrieval of supply chain statements
-5. Various Bill of Material Formats and metadata header
+
+1. make recommendations or suggestions on best practices on how to design the supply chain,
+2. establish a universal/centralized registry for supply chain data,
+3. try to prevent supply chain issuers from making false claims, 
+4. define specific implementation guidance on storage, query or retrieval of supply chain statements, or
+5. select specific Bill of Material formats and metadata header.
 
 Program of Work
 ===============
-The working group will develop standards supporting interoperable supply chain building blocks. The main deliverables are as follows:
+The working group will develop standards supporting interoperable supply chain building blocks.
+The main deliverables are as follows:
+
+1. The WG will select (and potentially profile or augment) acceptable common identity format/formats that shall be used in the SCITT eco-system. This will enable inter-operability across geographical regions and also tracing the identity chain in a seamless manner when a product is comprised of mutliple sub-products coming from multiple issuers. This includes a model of essential actors, such as the supply chain "issuer" (one which generates supply chain artefacts), and their duties in the ecosystem.
```

Comment:
```suggestion
1. The WG will select (and potentially profile or augment) acceptable common identity format/formats that shall be used in the SCITT eco-system. This will enable inter-operability across geographical regions and also tracing the identity chain in a seamless manner when a product is comprised of multiple sub-products coming from multiple issuers. This includes a model of essential actors, such as the supply chain "issuer" (one which generates supply chain artifacts), and their duties in the ecosystem.
```